scalafmt-core/shared/src/main/scala/org/scalafmt/util/TreeOps.scala

```diff
@@ -315,7 +315,7 @@ object TreeOps {
     tree match {
       case _: Decl.Def | _: Defn.Def | _: Defn.Macro | _: Defn.Class |
           _: Defn.Trait | _: Ctor.Secondary | _: Decl.Type | _: Defn.Type |
-          _: Type.Apply | _: Type.Param | _: Type.Tuple =>
+          _: Type.Apply | _: Type.Param | _: Type.Tuple | _: Defn.Enum =>
```

Comment:
I have found 5 invocations of `Defn.Class` in `TreeOps`, 2 more in `TokenOps`. Should we extend all those cases with `Defn.Enum`?

Reply:
yeah, I've also noticed these. I'm gonna push them